config/simulation/local.json

```
@@ -2,5 +2,6 @@
   "server": "localhost",
   "ssl": false,
   "portRewrite": false,
-  "debug": true
+  "debug": true,
+  "webotsHome": "PATH_TO_YOUR_WEBOTS_INSTALLATION"
```

I believe it's better to give a concrete example of something which may work (in most cases) rather that something which doesn't work. Users should be able to update this value to match their local installation.
```suggestion
  "webotsHome": "/usr/local/webots"
```